vep/src/main/resources/runVEP.sh

```diff
@@ -14,6 +14,13 @@ PATH="$PATH:$VEPDIR/samtools-1.9/:$VEPDIR/ensembl-vep/htslib"
 
 # copy the part file from S3 to local
 aws s3 cp "$S3DIR/variants/$PART" .
+extension="${PART##*.}"
+if [ "$extension" = "zst" ]
+then
+  unzstd "$PART"
+  rm "$PART"
+  PART="${PART%.zst}"
+fi
```

Comment:
It seems like Bash goes all in on being terse at the expense of telling you what the heck is going on.   I've tested this locally and it seems to get the value of whatever comes after the last . in PART.  If that value is zst, then decompress the file and re-assign PART so that .zst is removed from the end.

Reply:
Yup this looks good, I had to do something similar, although I couldn't tell you where 🤔